site/pages/react/using-7702.mdx

```diff
@@ -0,0 +1,70 @@
+- It is compatible with the prevailing AA standard, ERC-4437, so existing wallets can become smart contract wallets without creating a new address and transferring assets
+- For users, it will enable web2-like features, including account recovery and passkey sign-in for easier authentication, and web3-benefits like gas-sponsored transaction fees and paying for gas in any token, not just ETH
+
+[Learn more about EIP-7702](https://www.alchemy.com/blog/eip-7702-ethereum-pectra-hardfork#for-developers-that-havent-yet-deployed-their-application).
+
+## When to use EIP-7702?
+
+**For developers with existing users and embedded wallets**
+
+Upgrade your existing embedded EOA users to smart accounts to enable features like batching and gas sponsorship. Account Kit makes this upgrade easy with compatible smart accounts and infrastructure. Follow [this guide](#TODO/7702-uypgrade).
+
+**For developers who have not deployed their app**
+
+While ERC-4337 and EIP-7702 both enable account abstraction, they serve different needs in modern blockchain applications. You can either:
+
+- (Recommended) Use ERC-4337 smart accounts from the start: get started here
+- Use EIP-7702 smart EOAs: keep reading this guide
+
+See our full [recommendations](https://www.alchemy.com/blog/eip-7702-ethereum-pectra-hardfork#for-developers-that-havent-yet-deployed-their-application) to decide what's best for you.
+
+## How to use embedded EIP-7702 accounts
+
+In this example, we will upgrade a **new** embedded EOA to a smart contract account ([Modular Account v2](#TODO/link-toMAv2) specifically) and send a gasless transaction from the account.
+
+:::tip
+Note: Sepolia EIP-7702 support is not yet live. For early access to testnet support, please reach out to account-abstraction@alchemy.com
+:::
+
+The following sets up a Smart Account Client using the EIP-7702 supported version of a Modular Account v2. Make sure to first go through the [authentication setup](/react/getting-started) to generate a signer.
+
+```tsx twoslash
+import { useSmartAccountClient } from "@account-kit/react";
+
+const { client, address, isLoadingClient } = useSmartAccountClient({
+  type: "ModularAccountV2",
+  accountParams: {
+    mode: "7702",
+  },
+});
+```
+
+### 2. Send a sponsored transaction
+
+Now that you have a smart account client with an EIP-7702 supported smart account, you can leverage all of the easy-to-use React hooks to take action on the account. Account Kit will handle all of the complexity of delegating your EOA to the smart account and handling the special signature types.
+
+Continue on to [sending user operations](/react/send-user-operations) (aka smart account transactions) and [sponsoring gas](/react/sponsor-gas).
+
+## When can I use it?
+
+EIP-7702 will be supported on Sepolia starting March 5. The Alchemy infrastructure and Account Kit will have day-one support! Mainnet support is expected to follow shortly after.
```

Comment:
EIP-7702 will be supported on Sepolia starting March 5, and mainnet is expected to follow shortly after. Account Kit will support EIP-7702 integration on Day 1 at both milestones.